app/pages/sections/catalogue-solutions/order-item/template.test.js

```diff
@@ -23,6 +23,57 @@ describe('catalogue-solutions order-item page', () => {
     });
   }));
 
+  it('should render error summary with correct error text and hrefs if there are errors', componentTester(setup, (harness) => {
+    const context = {
+      questions: [
+        {},
```

Comment:
what do these empty objects refer to?

Reply:
OK I know now. These are the other questions we need to pass into the context. Cool this is fine 👍